lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenCompletionParticipant.java

```diff
@@ -328,11 +329,12 @@ public void onXMLContent(ICompletionRequest request, ICompletionResponse respons
 					.stream()
 					.map(artifacts -> Collections.max(artifacts, highestVersionWithDescriptionComparator))
 					.map(artifactInfo -> toGAVCompletionItem(artifactInfo, request, gavInsertionStrategy))
-					.forEach(response::addCompletionItem);
+					.forEach(completionItems::add);
 		}
 		if (request.getNode().isText() && (allArtifactInfos.isEmpty() || request.getNode().getTextContent().contains("$"))) {
 			completeProperties(request).forEach(response::addCompletionAttribute);
```

Comment:
I'm not sure what's the difference between a completion item and completion attribute? Maybe the properties should also be added to the set of completionItems.